src/h3lib/lib/h3Index.c

```diff
@@ -194,6 +194,8 @@ void H3_EXPORT(h3ToChildren)(H3Index h, int childRes, H3Index* children) {
     for (int i = 0; i < 7; i++) {
         if (!isAPentagon || i != K_AXES_DIGIT) {
             H3_EXPORT(h3ToChildren)(makeDirectChild(h, i), childRes, children);
+        } else {
+            *children = H3_INVALID_INDEX;
```

Comment:
I'm not sure this will zero enough of the buffer, since the recursive call to h3ToChildren could have generated more than one index. I think it needs to zero out `bufferChildStep` indexes.

Here is the test case I used:
```
TEST(pentagonChildrenMultires) {
    H3Index pentagon;
    setH3Index(&pentagon, 1, 4, 0);

    const int expectedCount = (5 * 7) + (6);
    const int paddedCount = H3_EXPORT(maxH3ToChildrenSize)(pentagon, 3);

    H3Index* children = calloc(paddedCount, sizeof(H3Index));
    H3_EXPORT(h3ToChildren)(sfHex8, 10, children);
    H3_EXPORT(h3ToChildren)(pentagon, 3, children);

    // just for testing:
    for (int i = 0; i < paddedCount; i++) {
        printf("test: %llx\n", children[i]);
    }

    verifyCountAndUniqueness(children, paddedCount, expectedCount);
    free(children);
}
```